### DISCUSSION TITLE
Why does pssIndices = np.arange((635-63), (635+64)) in the example. How did the values  635 come.

### DISCUSSION THREAD

**Discussion by amulamoottil** [catkira/py3gpp#32] [General]
Hi,
Why does pssIndices = np.arange((635-63), (635+64)) in the example have 635. How did the values  635 come. I know PSS is Mapped to 127 active sub carriers around the center of SSB block. But still cannot figure why the 635

**Comment by catkira** [comment]
thats because the ssb grid starts with ofdm symbol 0 but ssb starts at symbol 2

**Reply by amulamoottil** [reply]
Oh is there any data on the sigmf-data wave like how the waveform [30720KSPS_dl_signal.sigmf-data](https://github.com/catkira/py3gpp/blob/master/examples/30720KSPS_dl_signal.sigmf-data)  looks like and all or is there a way I can find that out. As it will be easier to figure the code out

**Reply by catkira** [reply]
yes, you can run the example scripts
